install/docker/index.markdown

```diff
@@ -29,54 +29,47 @@ If you are running a recent version of Ubuntu (e.g. 14.04, 16.04) it can be as s
 
 And you will likely need to log out and back into your user account for the changes to take affect.
 
-## Pulling MoveIt! Images
+## Running MoveIt! Containers
 
-Now that you have Docker installed, pull down a MoveIt! container image:
+### New Users
 
-    docker pull moveit/moveit
+To run a Debian-installed container of MoveIt! will graphics support:
 
-This will pull the latest tagged image of MoveIt! from Docker Hub onto your local host machine.
+    wget https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/.docker/gui-docker gui-docker
+    . ./gui-docker -ti --rm moveit/moveit:kinetic-release /bin/bash
 
-Within the [MoveIt! Dockerhub page](https://hub.docker.com/r/moveit/moveit/) you'll find additional tag names you can use to help specify what exact ROS version you'd like to use. For example:
+This will attempt to use nvidia-docker if hardware and drivers are available.
 
-    docker pull moveit/moveit:indigo-source
+**Command-Line Only**
 
-## Running MoveIt! Containers
+    docker run -it moveit/moveit:kinetic-release
 
-### New Users
+### Developers
 
-For a debian-installed container of MoveIt:
+To run a built-by-source container of MoveIt! will graphics support (using the Nvidia Docker container):
 
-    docker run -it moveit/moveit:kinetic-release
+    wget https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/.docker/gui-docker gui-docker
```

Comment:
Same comment as above. Also does this really need to be duplicated? Maybe just an aside that if you want to use containers with the gui use this separate script? Then just give the docker images for each use case developer/user.

Reply:
done